.github/workflows/build.yaml

```diff
@@ -12,6 +12,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          fetch-depth: "2"
```

Comment:
If this change doesn't work to remove `Unchanged files with check annotations` then remove this and try to find another solution that can work for it

Reply:
ok